src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -60,10 +60,11 @@ public class PreferenceSettings : NotificationObject, IPreferences, IRenderPreci
         /// <summary>
         /// Indicates whether usage reporting is approved or not.
         /// </summary>
+        [Obsolete("Property will be deprecated in Dynamo 3.0")]
         public bool IsUsageReportingApproved { get; set; }
 
         /// <summary>
-        /// Indicates whether analytics reporting is approved or not.
+        /// Indicates whether Google analytics reporting is approved or not.
         /// </summary>
         public bool IsAnalyticsReportingApproved { get; set; }
```

Comment:
At the moment the `IsAnalyticsReportingApproved ` property name does not match it's purpose.
I can do the following:
1. Obsolete it and add a new property IsGoogleAnalyticsReportingApproved.
2. Leave it as it is (name will not math descriptions/purpose)
3. Leave it as it was and add a new property IsGoogleAnalyticsReportingApproved - this way we can re-purpose it later on to represent the Parent Analytics Toggle (if we think we might do that)

Reply:
@pinzart Can we leave it as it is for now given the impact on migrating preference setting? I believe renaming this property, we will need to cover much edge cases and make sure people who use 1.3, 2.1 etc (which is a majority of our customer base) can migrate successfully.. I think we can do either 2 or 3. Since this particular PR is going to make it into 2.6.1, let's stick with 2 and once 2.6.1 is shipped, we can tackle option 3 with a new PR and only in newer Dynamo version

Reply:
Ok..sounds good to me.